index.html

```diff
@@ -0,0 +1,2110 @@
+            </path>
+          </g>
+        </svg>
+
+        <p class="copyright mt-3 ms-3">© 2021 NEWSWEEK DIGITAL LLC</p>
+      </div>
+      <div class="upper-footer-social-media d-inline d-md-flex align-items-end justify-content-md-end">
+        <a href="#"><i class="fab fa-facebook-square social-icons-footer"></i></a>
+        <a href="#"><i class="fab fa-twitter social-icons-footer"></i></a>
+        <a href="#"><i class="fab fa-instagram social-icons-footer"></i></a>
+        <a href="#"><i class="fab fa-tumblr social-icons-footer"></i></a>
+        <a href="#"><i class="fab fa-linkedin-in social-icons-footer"></i></a>
+      </div>
```

Comment:
- [ ] Please fix the color of the links so they are grey as in the original page and not the default link blue.
How the clone looks currently:
![Screenshot_2021-01-29 Newsweek](https://user-images.githubusercontent.com/16193212/106256465-13fd0780-61fa-11eb-82dc-50d81dc5ad97.png)